src/api/goods.ts

```diff
@@ -6,6 +6,16 @@ export function getAll(): Promise<Good[]> {
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = async () => {
+  const goods = await getAll();
 
-export const getRedGoods = () => {};
+  goods.sort((a, b) => a.name.localeCompare(b.name));
+
+  return goods.filter(good => (good.id <= 5));
```

Comment:
id doesn't mean that it is first product. After sorting you still check all products and choose it by id, not first 5